### DISCUSSION TITLE
vm ware

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS4719-C/Python-Community-GS4719-C#349] [General]
老師從第一堂課就開始使用vm ware(寫VM ware也可以)。我上vm ware的官網，想了解vm ware究竟是怎麼樣的東西。
目前由vm ware官網，得知對vm ware的理解如下：vm ware的功能是虛擬化(virtualization)。因為一臺電腦只能安裝一種且一個的作業系統，想要在不同作業系統的電腦作一樣的事情，就必須購買安裝各種作業系統的電腦主機。像我的電腦是windows作業系統，全世界還有mac OS(簡稱mac)與linux兩種作業系統。使用vm ware，可達到將作業系統虛擬化安裝在一臺電腦上的功能。這樣，我的電腦雖是windows作業系統，但我可以透過vm ware，將mac與linux這兩種作業系統，不安裝在我的電腦上，卻可以讓我使用這兩種作業系統。或許vm ware也可以讓我同時有好多windows作業系統的使用者。
![image](https://user-images.githubusercontent.com/86505454/150071370-5836c45d-7f37-45c8-886e-3e79248910ee.png)

**Comment by JunTingLin** [comment]
[enoch-lightning](https://github.com/enoch-lightning)同學介紹了VM的好VM的棒!!💪
 
 在此在補充一些


[💻童年之window XP(淺談虛擬機與操作)(原創文章) #370](https://github.com/NCU-GS4719-Python/Python-Community/discussions/370)

**Comment by handsomeniko** [comment]
很有幫助ㄟ，感謝分享。
